pypck/module.py

```diff
@@ -434,19 +477,20 @@ async def request_status_var_timeout(self, failed=False, var=None):
             PckGenerator.request_var_status(var, self.addr_conn.software_serial),
         )
 
-    def request_status_leds_and_logic_ops_timeout(self, failed=False):
+    def request_status_leds_and_logic_ops_timeout(self, failed: bool = False) -> None:
         """Is called on leds/logical ops status request timeout."""
         if not failed:
             self.addr_conn.send_command(
                 False, PckGenerator.request_leds_and_logic_ops()
             )
 
-    def request_status_locked_keys_timeout(self, failed=False):
+    def request_status_locked_keys_timeout(self, failed: bool = False) -> None:
         """Is called on locked keys status request timeout."""
         if not failed:
             self.addr_conn.send_command(False, PckGenerator.request_key_lock_status())
 
-    async def activate(self, item):
+    @typing.no_type_check
+    async def activate(self, item: Any) -> None:
```

Comment:
You are using `Any` as a type hint for `item` (which basically is a Union of Enums).
Thus the `@no_type_check` decorator shouldn't be necessary?

Reply:
You are right, the marker is not needed. I think I added them initially because I ran into problems with mypy and this function. Eventually settled for `Any` but I did not check if the marker is necessary. It's now removed.